practices/dependency-management.md

```diff
@@ -0,0 +1,104 @@
+# Dependency management
+
+The following guide describes how we manage package dependencies in our Node.js-based projects.
+
+
+## Specifying versions of dependencies
+
+As a general rule, we tend to follow a conservative approach when specifying versions for the dependencies of our packages. This helps us ensure consistency between different environments, as we can't know for sure in what environment the app is going to be running, and prevent breakage caused by major or minor updates in dependencies.
+
+We find that the this approach strikes a good balance between the potential breakage caused by less restrictive versions and the maintenance needed when restricting the dependencies to a specific version through the use of `npm shrinkwrap` or similar tools.
+
+We use some [dependency management tools](#dependency-management-tools) to help us keep our apps up to date when new versions of their dependencies are released.
+
+
+### Run-time dependencies
+
+[Run-time dependencies](https://docs.npmjs.com/files/package.json#dependencies) are modules that are required for your application to run, independently of the environment or mode (e.g. development vs production) used.
+
+These are defined in the `dependencies` section of the `package.json` file.
+
+Version numbers for run-time dependencies are defined using a tilde `~` plus a full version number in the MAJOR.MINOR.PATCH form. If the PATCH version is 0, it can be omitted.
+
+Specifying the versions in this way ensures that:
+* We don't expose the users of our modules to an unnecessary risk if one of our dependencies releases a minor update that breaks our app.
+* Dependencies in early lifecycle projects (v0.MINOR.PATCH) are traditionally considered of beta quality, or undergoing heavy development, and any minor could potentially contain breaking changes. Limiting the scope to patch updates heavily reduces the chance of our app breaking.
+* The behaviour when installing the dependencies is basically as predictable as possible ([Principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)) without the use of npm-shrinkwrap or similar tools.
+
+Don't use wildcards (i.e. `*`, `x`) in the package numbers or keywords like `latest`. They make the versions less clear, and predicting the outcome of installing the dependencies harder.
```

Comment:
"predicting the outcome of installing the dependencies harder" makes my brain hurt. Is it possible to reword, somehow? Maybe using commas, or … I dunno. Not sure how. Work some magic with the English language.